internal/web/templates/components/pkg-config-advanced.html

```diff
@@ -0,0 +1,88 @@
+{{ define "pkg-config-advanced" }}
+  <div class="mt-3 border-warning border border-1 rounded-1 p-2 bg-body-secondary" id="advanced-options">
+    <h2 class="text-reset">Advanced Options</h2>
+    <div class="alert alert-warning" role="alert">
+      <i class="bi bi-exclamation-triangle-fill text-warning me-1"></i>Changing the repository or downgrading the
+      package version is not recommended and should never be done except in dev/test environments. This could lead to
+      data loss and a broken cluster! <b>Use at your own risk!</b>
+    </div>
+
+    <form
+      hx-post="/packages/{{ .Manifest.Name }}/configure/advanced"
+      hx-target="#error-container"
+      hx-target-error="#error-container"
+      hx-select="div.alert"
+      hx-swap="innerHTML">
+      {{ if gt (len .Repositories) 1 }}
+        <label class="form-label" for="advanced-pkg-install-repository">Repository</label>
+        <div class="input-group mb-2">
+          <select
+            class="form-select"
+            id="advanced-pkg-install-repository"
+            name="repositoryName"
+            required
+            hx-select="#advanced-options"
+            hx-swap="outerHTML"
+            hx-target="#advanced-options"
+            hx-get="/packages/{{ .Manifest.Name }}/configure/advanced"
+            hx-include="#advanced-pkg-install-version">
+            {{ range .Repositories }}
+              <option value="{{ .Name }}" {{ if eq .Name $.RepositoryName }}selected{{ end }}>
```

Comment:
I noticed that the current repository is not selected initially, even though it should be, according to this.